### PR TITLE
Provide an entry page for GraalVM CE container images in GitHub Registry

### PR DESCRIPTION
**Pull request by olyagpl**

### CODE REVIEW

**Review by ezzarghili — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+You can find a complete GraalVM Community image, a compact image without additional features, or GraalVM Community image with specific features like the Node.js runtime.
+
+The images are tagged with the format `ghcr.io/OWNER/IMAGE_NAME:version`, where `OWNER` is the repository name, `graalvm` in our case, and `IMAGE_NAME` is the package name.
+The version tag defines the level of specificity.
+It is recommended to use the most specific tag, e.g., `java17-21.3.0` or `java17-21.3.0-b1`, where the `_b1` means the image required a patch and was rebuilt.
+
+You can find the following packages available:
+
+| Package      | Description                                                                                                                                  |
+|--------------|----------------------------------------------------------------------------------------------------------------------------------------------|
```
[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md:16]: Good to know!  My rewrote:
_Some images are `gu`-based (containing the [GraalVM Updater](https://github.com/oracle/graal/blob/master/docs/reference-manual/graalvm-updater.md) tool), some are RPM-based._

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+To support container-based development, GraalVM Community container images are published in the GitHub Container Registry for each release.
+
+There are different GraalVM Community container images provided depending on the architecture and the Java version.
+The images are multi-arch (aarch64 or amd64 depending on the host architecture).
+You can find a complete GraalVM Community image, a compact image without additional features, or GraalVM Community image with specific features like the Node.js runtime.
+
+The images are tagged with the format `ghcr.io/OWNER/IMAGE_NAME:version`, where `OWNER` is the repository name, `graalvm` in our case, and `IMAGE_NAME` is the package name.
+The version tag defines the level of specificity.
+It is recommended to use the most specific tag, e.g., `java17-21.3.0` or `java17-21.3.0-b1`, where the `_b1` means the image required a patch and was rebuilt.
```
[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md:11]: _It is recommended to use the most specific tag, e.g., `java17-21.3.0` or `java17-21.3.0-b1`, where the 
`-b1` means the image required a patch and this specific build will never change._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# GraalVM Community Container Images
+
+To support container-based development, GraalVM Community container images are published in the GitHub Container Registry for each release.
+
+There are different GraalVM Community container images provided depending on the architecture and the Java version.
+The images are multi-arch (aarch64 or amd64 depending on the host architecture).
+You can find a complete GraalVM Community image, a compact image without additional features, or GraalVM Community image with specific features like the Node.js runtime.
+
+The images are tagged with the format `ghcr.io/OWNER/IMAGE_NAME:version`, where `OWNER` is the repository name, `graalvm` in our case, and `IMAGE_NAME` is the package name.
```
[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md:9]: _The images are tagged with the format `ghcr.io/graalvm/IMAGE_NAME:version`, where `IMAGE_NAME` is a package name._

I would explain just why image name = package name, what you see on https://github.com/orgs/graalvm/packages?repo_name=container.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+It is recommended to use the most specific tag, e.g., `java17-21.3.0` or `java17-21.3.0-b1`, where the `_b1` means the image required a patch and was rebuilt.
+
+You can find the following packages available:
+
+| Package      | Description                                                                                                                                  |
+|--------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| jdk          | GraalVM Community compact JDK container image without additional features.                                                                   |
+| graalvm-ce    | GraalVM Community Edition image with GraalVM Updater which allows to install and manage additional features like Native Image, Node.js, etc. |
+| native-image |  GraalVM Community compact image with Native Image.                                                                                          |
+| truffleruby  | GraalVM Community compact image with the Ruby runtime.                                                                                       |
```
[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md:20]: Done

File: README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+You can pull a package by name or by name and version tag.
+To install from the command line, use:
+```
+$ docker pull ghcr.io/graalvm/jdk:java17-21.3.0-b1
+```
+To use GraalVM JDK as base image in Dockerfile, use:
+```
+FROM ghcr.io/graalvm/jdk:java17-21.3.0-b1
+```
+
```
[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md:32]: Thanks!

**Review by ezzarghili — APPROVED** [review]

**Review by shaunsmith — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+# GraalVM Community Container Images
```
[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md:1]: I tried to be less wordy, but according to our guidelines, we indeed should use a full name, "GraalVM Community Edition".

File: README.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+| native-image |  GraalVM Community compact image with Native Image.                                                                                                        | RMP-based |
+| truffleruby  | GraalVM Community compact image with the Ruby runtime. The image uses a standalone build of [TruffleRuby](https://github.com/oracle/truffleruby/releases). | RMP-based |
+| nodejs       | GraalVM Community compact image with the Node.js runtime.                                                                                                  | RMP-based |
+
+The images are tagged with the format `ghcr.io/graalvm/IMAGE_NAME:version`, where `IMAGE_NAME` is a package name.
+The version tag defines the level of peculiarity.
+It is recommended to use the most specific tag, e.g., `java17-21.3.0` or `java17-21.3.0-b1`, where the
+`-b1` means the image required a patch and this specific build will never change.
+
+You can pull a package by name or by name and version tag.
```
[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md:24]: Got it

File: README.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+There are different GraalVM Community container images provided depending on the architecture and the Java version.
+The images are multi-arch (`aarch64` or `amd64` depending on the host architecture).
+Some images are `gu`-based (containing the [GraalVM Updater](https://github.com/oracle/graal/blob/master/docs/reference-manual/graalvm-updater.md) tool), some are RPM-based.
+You can find complete images, compact images without additional features, or images with specific features included.
+All images allow to install extra features. You can find the following packages available:
+
+| Package      | Description                                                                                                                                                | Type      |
+|--------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| jdk          | GraalVM Community compact JDK container image without additional features.                                                                                 | RMP-based |
```
[the remark this answers is not in the document]
Comment (ezzarghili) [reply README.md]: I agree.
For instance:

> JDK: A size compact GraalVM Community container image with the GraalVM JDK [rpm](https://yum.oracle.com/repo/OracleLinux/OL7/graalvm/community/x86_64/index.html) pre-installed.


maybe use a reference here such as  RMP-based (1)

> (1) Bases on GraalVM components rpms available for [Oracle Linux 7](https://yum.oracle.com/repo/OracleLinux/OL7/graalvm/community/x86_64/index.html)  or  [Oracle Linux 8](https://yum.oracle.com/repo/OracleLinux/OL8/graalvm/community/x86_64/index.html),  similar to any other available packages, you can install these packages using `yum` on OL7 or `microdnf` on the OL8 based images.

[the remark this answers is not in the document]
Comment (ezzarghili) [reply README.md]: @olyagpl I will leave it to you to find better wording for this.

[the remark this answers is not in the document]
Comment (olyagpl) [reply README.md]: @shaunsmith, please review https://github.com/graalvm/container/pull/32/commits/7feeba41151d71a395262984efd704869f719e12

**Pull request merged**